t/run/locale.t

```diff
@@ -354,7 +354,8 @@ EOF
 
         {
             open my $saved_stderr, ">&STDERR" or die "Can't dup STDERR: $!";
-            close STDERR;
+            # Comment out the following line to get error output when running the test
+            # close STDERR;
```

Comment:
Going by your comments, you shouldn't have commented this line out.

Reply:
Opps, you are right. Sorry, fixed now.